PythonMetricsCalculator/PythonMetricsCalculator.py

```diff
@@ -1,6 +1,7 @@
 import os, imp, glob, sys
 import urllib, zipfile
-import urllib.request
+if sys.version_info.major == 3:
+  import urllib.request
```

Comment:
I wonder if you do something like:

```python
try:
  from urllib.request import urlretrieve # Python 3
except ImportError:
  from urllib import urlretrieve # Python 2
```

Then you could avoid the extra conditional below, reducing the amount of version-specific code while keeping compatible with both Python 2 and Python 3.